File: developer-experience/developer-experience-native-android/app/src/main/java/com/iman/jamali/crud_native_android/data/model/Todo.java
package com.iman.jamali.crud_native_android.data.model;

import com.google.gson.annotations.Expose;

public class Todo {
    private final String id;
    @Expose
    private final String title;
    @Expose
    private final String description;
    private final String updatedAt;

    public Todo(String id, String title, String description, String updatedAt) {
        this.id = id;
        this.title = title;
        this.description = description;
        this.updatedAt = updatedAt;
    }

    public String getId() {
        return id;
    }
    public String getTitle() {
        return title;
    }
    public String getDescription() {
        return description;
    }
    public String getUpdatedAt() {
        return updatedAt;
    }
}

File: developer-experience/developer_experience_flutter/lib/todo.dart
class Todo {
  final String id;
  final String title;
  final String description;
  final String updatedAt;
  
  Todo({required this.id, required this.title, required this.description,       
  required this.updatedAt});
  
  factory Todo.fromJson(Map<String, dynamic> json) {
    return Todo(
      id: json['id'],
      title: json['title'],
      description: json['description'],
      updatedAt: json['updatedAt']
    );
  }

  Map<String, dynamic> toJson() => {
    'title': title,
    'description': description,
  };
}